tools/generator/xcodeproj_targets.bzl

```
@@ -8,7 +8,7 @@ GENERATOR_SCHEME_AUTOGENERATION_MODE = "all"
 def get_xcode_schemes():
     return [
         xcode_schemes.scheme(
-            name = "Generator App",
+            name = "Generator",
```

I know it doesn't exercise the normalization logic, but I really want this scheme to be named after the product, which is lowercase.

```suggestion
            name = "generator",
```

---

Done